Enhance README.md (#10)
vtex-apps/example-payment-authorization-app: README.md

`http://{workspace}--{account}.myvtex.com/checkout?workspace={workspace}&orderFormId={orderFormId}`.

Once you're happy with the changes, follow our documentation on [making your new app version publicly available](https://vtex.io/docs/recipes/development/making-your-new-app-version-publicly-available/) to run your app on master.

## Modus Operandi

After developing and implementing the *Payment* app, your store will profit from having an additional verification step in its checkout process.

To make the best use of the *Payment* app, this section discusses the following topics:

- Understanding the checkout response.
- Handling the order payload.
- Injecting external scripts.

### Understanding the checkout response

After the order placement, there's a validation step to check whether the transaction is approved or not.

If the transaction is approved, the checkout UI redirects to the *Order Placed* page. Otherwise, it shows a warning pop-up.

> This validation is performed through the `transactionValidation.vtex` event of our *Payment* app, which sends a response to the checkout UI according to the following [method](https://github.com/vtex-apps/payment-authorization-app-example/blob/3e5742c87a2771998009cff4fecacb092bb3362b/react/index.js#L22):

Author: Acho que faria mais sentido algo assim:

> ... app, which sends the status of payment to the checkout UI ...

> This validation is performed through the `transactionValidation.vtex` event of our *Payment* app, which sends a response to the checkout UI according to the following [method](https://github.com/vtex-apps/payment-authorization-app-example/blob/3e5742c87a2771998009cff4fecacb092bb3362b/react/index.js#L22):

Reviewer: ```suggestion
In our *Payment* app, this validation is performed through the `transactionValidation.vtex` event, which, when triggered, sends the payment status to the checkout UI according to the following [method](https://github.com/vtex-apps/payment-authorization-app-example/blob/3e5742c87a2771998009cff4fecacb092bb3362b/react/index.js#L22):
```

```javascript
respondTransaction = (status) => {
    $(window).trigger('transactionValidation.vtex', [status])
}
```

Notice that the `respondTransaction` function receives the `status` boolean value, which resolves (`status == true`) or rejects (`status == false`) an order placement.

> Hence, according to the `status` value, whenever the `transactionValidation.vtex` event occurs, the `respondTransaction` function will send the appropriate response to the checkout UI.

Author: Acho que faria mais sentido algo assim:

> `transactionValidation.vtex` event occurs, the Checkout UI will check the appropriate payment status.

> Hence, according to the `status` value, whenever the `transactionValidation.vtex` event occurs, the `respondTransaction` function will send the appropriate response to the checkout UI.

Reviewer: ```suggestion
Hence, whenever the `transactionValidation.vtex` event occurs, the Checkout UI checks the payment status and is redirected according to the `status` value provided to the `respondTransaction` function
```

### Handling the order payload

The `appPayload` is what the *Payment* app receives as `props` and it consists of a serialized JSON of the order payload.

Thus, to access this information, just run the following command:

```javascript
const { appPayload } = this.props // This appPayload is a serialized JSON (as string).
```

The expected response is a JSON object as in the following:

```json

> ```

Reviewer: @igorbrasileiro tem um exemplo para a gente colocar aqui? 🙈

In our *Payment* app, this validation is performed through the `transactionValidation.vtex` event, which, when triggered, sends the payment status to the checkout UI according to the following [method](https://github.com/vtex-apps/payment-authorization-app-example/blob/3e5742c87a2771998009cff4fecacb092bb3362b/react/index.js#L22):

```javascript
respondTransaction = (status) => {
    $(window).trigger('transactionValidation.vtex', [status])
}
```

Notice that the `respondTransaction` function receives the `status` boolean value, which resolves (`status == true`) or rejects (`status == false`) an order placement.

Hence, whenever the `transactionValidation.vtex` event occurs, the Checkout UI checks the payment status and is redirected according to the `status` value provided to the `respondTransaction` function

### Handling the order payload

The `appPayload` is what the *Payment* app receives as `props` and it consists of a serialized JSON of the order payload.

Thus, to access this information, just run the following command:

```javascript
const { appPayload } = this.props // This appPayload is a serialized JSON (as string).
```

> The expected response is a JSON object as in the following:

Reviewer: ```suggestion
The expected response is a JSON object consisting of the data needed for the Payment app to allow or deny an order placement. Thus, depending on the payment processes implemented in your store, the `appPayload` fields may vary.

Take the following example:
```

Reviewer: @igorbrasileiro faz sentido? KK

Author: Faz

```json
{
    "timeToCancel": 300,

>     "transactionApproveLinl": "https://..."

Author: ```suggestion
    "transactionApproveLink": "https://..."
```
